### PR TITLE
Coordinates from coordgen are not centered around the origin

### PR DESCRIPTION
**Pull request by DavidACosgrove**
When using coordGen for 2D coords, always moves the coords to be centred on (0,0).  Fixes an issues laying out some molecules in grids.

Tested on Mac only.

### CODE REVIEW

File: Code/GraphMol/Depictor/RDDepictor.cpp [hunk 1]
```diff
@@ -315,7 +317,15 @@ unsigned int compute2DCoords(RDKit::ROMol &mol,
     if (coordMap) {
       params.coordMap = *coordMap;
     }
-    return RDKit::CoordGen::addCoords(mol, &params);
+    unsigned int cid = RDKit::CoordGen::addCoords(mol, &params);
+    RDGeom::Point3D centroid = MolTransforms::computeCentroid(mol.getConformer(cid));
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/RDDepictor.cpp:321]: We shouldn't be calculating the centroid and doing the coordinate shift if `coordMap` is provided.

File: Code/GraphMol/MolTransforms/test1.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@
 #include <GraphMol/SmilesParse/SmilesParse.h>
 #include <GraphMol/FileParsers/FileParsers.h>
 #include <Geometry/point.h>
-#include "MolTransforms.h"
+#include <GraphMol/MolTransforms/MolTransforms.h>
```
Comment (greglandrum) [inline Code/GraphMol/MolTransforms/test1.cpp:19]: Why is this needed here?

File: Code/GraphMol/Depictor/RDDepictor.cpp [hunk 3]
```diff
@@ -315,7 +317,15 @@ unsigned int compute2DCoords(RDKit::ROMol &mol,
...
+    RDGeom::Transform3D trans;
+    trans.SetTranslation(-centroid);
+    RDGeom::POINT3D_VECT &locs = mol.getConformer(cid).getPositions();
+    for(auto atom: mol.atoms()) {
+      trans.TransformPoint(locs[atom->getIdx()]);
+    }
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/RDDepictor.cpp:327]: Can be tons simpler:
```suggestion
    auto &conf = mol.getConformer(cid);
    MolTransforms::transformConformer(conf,trans);
```

Comment (DavidACosgrove) [reply Code/GraphMol/Depictor/RDDepictor.cpp:327]: I thought there must be such  thing, but couldn't find it.

File: Code/GraphMol/MolTransforms/test1.cpp (hunk 2 above)
Comment (DavidACosgrove) [reply Code/GraphMol/MolTransforms/test1.cpp:19]: When I went looking for examples of MolTransforms, this is the first place I found it referenced. Then I had to do another find for MolTransforms.h.  Since it's part of the RDKit API, it seemed tidier to include it using the same syntax as all the other bits.

Comment (greglandrum) [reply Code/GraphMol/MolTransforms/test1.cpp:19]: ah, sorry. I missed that this was changing an existing `#include`

**Comment by greglandrum** [comment]
Thanks @DavidACosgrove. Will merge after the CI builds pass
